README.md

```diff
@@ -107,6 +108,7 @@ A curated list of awesome research papers, datasets and software projects devote
 
 #### Source Code Analysis and Language modeling
 
+- <img src="badges/12-pages-gray.svg" alt="24-pages" align="top"> [Modeling Vocabulary for Big Code Machine Learning](https://arxiv.org/abs/1904.01873v1) - Hlib Babii, Andrea Janes, Romain Robbes, 2019.
```

Comment:
```suggestion
- <img src="badges/12-pages-gray.svg" alt="12-pages" align="top"> [Modeling Vocabulary for Big Code Machine Learning](https://arxiv.org/abs/1904.01873v1) - Hlib Babii, Andrea Janes, Romain Robbes, 2019.
```

Reply:
completely forgot about a second place with a number of pages - nicely spotted)